test/trivial_trading.js

```diff
@@ -0,0 +1,97 @@
+var common = require('./trivial_tests_common.js');
+var TrivialToken = artifacts.require("TrivialToken.sol");
+
+contract('TrivialToken - Exchange tests', (accounts) => {
+    var token;
+    var trivialAddress = accounts[0];
+    var artistAddress = accounts[1];
+    var userAddress2 = accounts[2];
+    var userAddress3 = accounts[3];
+    var userAddress4 = accounts[4];
+
+    async function contributeInIco() {
+        assert.equal(await token.balanceOf.call(userAddress4), 0,
+            'Should be zero before contribution');
+        await token.contributeInIco({from: userAddress2, value: web3.toWei(2, 'ether')});
```

Comment:
Fails due to `onlyBefore(icoEndTime)` and some issue connected to `evm.ForwardTime`